### DISCUSSION TITLE
怎么样禁止shell的自动完成？

### DISCUSSION THREAD

**Discussion by wsw0108** [xmake-io/xmake#2544] [Q&A]
比如在`xmake r target /path/to| `时，`|`是光标位置，tab总是补全xmake的子命令，而不是文件路径。

**Comment by waruqi** [comment]
~/.xmake/profile 里面  `_xmake_bash_complete` 相关代码删了

**Reply by wsw0108** [reply]
Windows下，.xmake(AppData/Roaming/.xmake)下没有profile文件。
```
cat .\AppData\Roaming\.xmake\xmake.conf
{
    pkg_installdir = "D:/packages/xmake",
    proxy_pac = "pac.lua",
    theme = "default",
    network = "public",
    __version = "2.6.8"
}
```

**Reply by waruqi** [reply]
win 上只有走一键脚本安装，才会装 shell 完成，并且安装过程有提示，你可以选择不装的

https://github.com/xmake-io/xmake/blob/e8911871580e32e4b782c2bc98ed15248839916f/scripts/get.ps1#L167-L186

重装下就行了，另外，releases 下的安装包，默认安装是不会装的，你可以走安装包重新安装

**Reply by wsw0108** [reply]
好的，知道原因了，目前使用scoop安装的，以前的一个$PROFILE.CurrentUserAllHosts还存在。。。

**Comment by wsw0108** [comment]
https://github.com/xmake-io/xmake/discussions/2544#discussioncomment-3104316
